core/server/services/members/api.js

```diff
@@ -152,7 +152,15 @@ function createApiInstance() {
                     if (process.env.NODE_ENV !== 'production') {
                         common.logging.warn(message.text);
                     }
-                    return ghostMailer.send(Object.assign({subject: 'Signin'}, message));
+
+                    let msg = Object.assign({subject: 'Signin'}, message);
+                    const subscriptionSettings = settingsCache.get('members_subscription_settings');
+
+                    if (subscriptionSettings && subscriptionSettings.fromAddress) {
+                        msg = Object.assign({from: subscriptionSettings.fromAddress}, msg);
```

Comment:
I believe this should be something like `from: `fromAddress@${getDomainOfTheSite()}`

Reply:
@allouis fixed the email construction code. Don't have the flow running e2e at the moment, would appreciate a quick real-world check before merge ;)